server/metrics/relay.go

```diff
@@ -0,0 +1,132 @@
+package metrics
+
+import (
+	"context"
+	"github.com/jitsucom/jitsu/server/resources"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/carlmjohnson/requests"
+	"github.com/jitsucom/jitsu/server/logging"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+var HashedRelayLabels = map[string]bool{
+	"source_id":      true,
+	"destination_id": true,
+}
+
+type RelayTrigger interface {
+	Channel() <-chan time.Time
+	Now() time.Time
+	Stop()
+}
+
+type TickerTrigger struct {
+	*time.Ticker
+}
+
+func (t TickerTrigger) Now() time.Time {
+	return time.Now()
```

Comment:
Please avoid using `time.Now()` - use custom wrapper instead: `timestamp.Now()`. It's a wrapper for mocking time in tests.

Reply:
Fixed.